writeup/conStruct.tex

```diff
@@ -1650,18 +1650,17 @@ \section*{Cross validation}\label{Xvalidation}
         95\% confidence interval ($\bar{\mathcal{Z}}_{mk} \pm 1.96 \times SE_{\bar{\mathcal{Z}}_{mk}}$).
 \end{enumerate}
 
-\gb{If the spatial coordinates of the loci in the genome are known, 
+If the genomic coordinates of the loci are known, 
 the training/testing partitioning should be designed to accommodate linkage disequilibrium (LD).
-Loci in strong LD do not represent independent instantiations of the coalescent process, 
-so if loci from a single linkage block are randomly split into both training and testing partitions, 
-the independence of the test in the testing partition will be compromised 
-(e.g., the parameters estimated from the training partition 
```

Comment:
this is a relevant point even without bringing the coalescent into it (although that's good intuition)